src/services/anchor-service.ts

```diff
@@ -317,7 +318,7 @@ export default class AnchorService {
         const candidate = new Candidate(new CID(request.cid), request.id, doc);
         groupedCandidates[candidate.docId] = groupedCandidates[candidate.docId] ? [...groupedCandidates[candidate.docId], candidate] : [candidate];
       } catch (e) {
-        logger.err(e);
+        logger.err(`Error while loading document ${docId.baseID.toString()} at commit ${docId.commit.toString}. Error: ${e.message}`)
```

Comment:
This message breaks if docId is null/undefined. We should probably throw in that case or have an additional try/catch for it above.

Reply:
Just add some `?`